docs/tutorials/go/stripe-integration.mdx

```diff
@@ -0,0 +1,427 @@
+## Introduction
+
+This tutorial will help you get set up and running with your first integration with Stripe - a popular payments gateway.
+The code snippets in this tutorial are lightly modified versions of the Stripe QuickStart featuring React and Go.
+
+## Requirements
+
+- Confirm that you have Node installed with the following command `node --version`
+- It is also assumed that you have already installed the Nitric CLI. If not visit the [installation instructions](../installation.mdx).
+
+## Project set up
+
+For this project we will make a simple react app which redirect to the Stripe Payments Gateway.
+
+Using the Nitric CLI we start by creating the project.
+
+```bash
+nitric make:project nitric-stripe-payments
+```
+
+Next we'll be asked if we want an initial service in the project and what template the service will use.
+For this project we will want a service. This service will handle requests for greeting and respond to the greetings. For this we will use the `nodets12` template.
+
+x
+? Include an example service?
+function/nodets12
+function/nodejs12
+function/python37
+function/java11
```

Comment:
Could we move this tutorial to a separate PR?

The troubleshooting and tutorials are independently valuable.

Reply:
Yup, I'll separate it - slipped in